api/circulation.py

```diff
@@ -95,11 +94,28 @@ def __repr__(self):
         )
 
 
+
+class PatronInfo(CirculationInfo):
+    """ Hold patron account data coming back from OneClick before we 
+    merge it into our database. """
+    def __init__(self, oneclick_id=None, username=None, card_number=None, card_pin=None, 
+        email=None, first_name=None, last_name=None):
+        self.oneclick_id = oneclick_id
+        self.username = username
+        self.card_number = card_number
+        self.card_pin = card_pin
+        self.email = email
+        self.first_name = first_name
+        self.last_name = last_name
+
```

Comment:
Do you plan to use this information? Generally we get this stuff from the ILS. I see that this is created by `get_patron_information` but I'm not sure when we would need that information. If One Click has its own ID for patrons I could see storing that as a Credential associated with the Patron, but I feel like you cover that case with `get_patron_internal_id`.

If we need to use this information I'd suggest subclassing the new PatronData class (in authenticator.py) and storing it in there. But unless you have a use case in mind I'd say leave it alone until we determine whether we actually need it.